src/main/java/no/sikt/nva/pubchannels/channelregistrycache/ChannelRegistryCacheEntry.java

```diff
@@ -0,0 +1,130 @@
+package no.sikt.nva.pubchannels.channelregistrycache;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.opencsv.bean.CsvBindByName;
+import java.util.Arrays;
+import java.util.List;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.core.JacocoGenerated;
+
+public class ChannelRegistryCacheEntry {
```

Comment:
Why add data we don't use? You can remove "DOAJ", "Publiseringsavtale", "Internasjonal tittel" and "regDato"

Reply:
I do not know yet what we will be using, but if you sure, I remove it.

Reply:
You can see what we use today in `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryJournal`, `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryPublisher` and `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistrySeries`